src/psyclone/tests/psyir/frontend/size_intrinsic_test.py

```diff
@@ -45,9 +45,14 @@
 
 @pytest.mark.parametrize("expression", ["n = SIZE(a, 3)",
                                         "n = SIZE(a(:,:,:), 3)"])
+@pytest.mark.usefixtures("disable_declaration_check")
 def test_size(expression, parser):
     ''' Basic test that the SIZE intrinsic is recognised and represented
-    in the PSyIR. '''
+    in the PSyIR.
```

Comment:
pylint:
W: 49,26: Unused argument 'parser' (unused-argument)
Change to the other form of initialising the parser ... @pytest.mark.usefixture ...?

Reply:
Good spot, done.